liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -403,7 +403,15 @@ public void checkConfigFile(File configFile, File serverDir) {
                     features.removeAll(existingFeatures);
                     if (!features.isEmpty()) {
                         log.info("Configuration features have been added");
-                        runLibertyMojoInstallFeature();
+                        Element[] featureElems = new Element[features.size() + 1];
+                        int i = 0;
+                        for (String f : features) {
+                            featureElems[i] = element(name("feature"), f);
+                            i++;
+                        }
+                        featureElems[i] = element(name("acceptLicense"), "true");
+
+                        runLibertyMojoInstallFeature(element(name("features"), featureElems));
```

Comment:
Quoted from #620
> Since liberty:install-feature is called before the changed server.xml is copied to the target, the new features added to the server.xml is not installed, Change the code to call install-features with the list of new features when source server.xml is changed

This doesn't seem like the root cause.  The server.xml (and the entire server dir) is supposed to be copied to a temp dir when changes are detected (see `DevUtil.copyConfigFolder()` which then calls `checkConfigFile()`), which then should install the features from the server.xml from the temp dir.  The configuration list of features should not need to be created in the pom.

Reply:
This is from the original implementation:
https://github.com/OpenLiberty/ci.maven/blob/0e40c5ff4378e8714c2053db940e46c087136281/liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java/#L395:L413.